libs/langchain-google-common/src/auth.ts

```diff
@@ -58,15 +73,18 @@ export abstract class GoogleAbstractedFetchClient
         `Google request failed with status code ${res.status}: ${resText}`
       );
       // eslint-disable-next-line @typescript-eslint/no-explicit-any
-      (error as any).response = res;
+      (error as any).details = {
```

Comment:
Good to leave `response` directly on the error object - `AsyncCaller` uses fields on it to decide retries.